it/SelectNationalInsuranceServiceControllerISpec.scala

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import play.api.Application
+import play.api.http.Status.{OK, SEE_OTHER}
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.ws.DefaultWSCookie
+import play.api.test.Helpers.LOCATION
+
+class SelectNationalInsuranceServiceControllerISpec extends HelperSpec {
+
+  val getPageBaseUrl = "/helpline"
+  val selectNationalInsuranceServiceKey = "/select-national-insurance-service"
+  val nationalInsuranceHelpline = "/NATIONAL-INSURANCE"
+
+  override lazy val app: Application = new GuiceApplicationBuilder()
+    .configure(
+      "play.filters.csrf.header.bypassHeaders.X-Requested-With" -> "*",
+      "play.filters.csrf.header.bypassHeaders.Csrf-Token" -> "nocheck"
+    )
+    .build()
+
+
+  "GET /select-national-insurance-service" should {
+    "redirect to the select national insurance service page" in {
+      withClient {
+        wsClient => {
+        val submitNationalInsuranceServiceResponse = wsClient.url(resource(s"$getPageBaseUrl$selectNationalInsuranceServiceKey")).get().futureValue
+          submitNationalInsuranceServiceResponse.status shouldBe OK
+        }
+      }
```

Comment:
nice, could you possibly add an IT test, using this GET as an example so we have it covered (could be added to this test)
- Referer provided in header
- Session value set for the helpline origin